src/index.html

```diff
@@ -11,7 +11,171 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="form-box">
+        <legend>Personal information</legend>
+
+        <div class="form-field">
+          <lable>
+            Surname:
+            <input
+            type="text"
+            name="surname"
+            autocomplete="off"
+            >
+          </lable>
+        </div>
+
+        <div class="form-field">
+          <lable>
+            Name:
+            <input
+            type="text"
+            name="name"
+            autocomplete="off"
+            >
+          </lable>
+        </div>
+
+        <div class="form-field">
+          <lable>
+            How old are You?
+            <input
+            type="number"
+            name="age"
+            min="1"
+            max="100"
+            value="12"
+            >
+          </lable>
+        </div>
+
+        <div class="form-field">
+          <lable>
+            Full date of birth:
+            <input type="date" name="dateOfBirth">
+          </lable>
+        </div>
+
+        <div>
+          <lable>
+            I accept the term of agreement
+            <input
+            type="checkbox"
+            name="termCheck"
+            required
+            >
+          </lable>
+        </div>
+      </fieldset>
+
+      <fieldset class="form-box">
+        <legend>Registration</legend>
+
+        <div class="form-field">
+          <label>
+            E-mail:
+            <input
+            type="email"
+            name="email"
+            placeholder="email@example.com"
+            >
+          </label>
+        </div>
+
+        <div>
+          <label>
+            Password:
+            <input
+            type="password"
+            name="password"
+            minlength="8"
+            maxlength="100"
+            >
+          </label>
+        </div>
+      </fieldset>
+
+      <fieldset class="form-box">
+        <legend>An interesting fact about you!</legend>
+
+        <div class="form-field">
+          <label>
+            Do you have cats?
+            <input
+            type="radio"
+            name="pets"
+            value="yes"
+            >
+            Yes
+
+            <input
+            type="radio"
+            name="pets"
+            value="no"
+            >
+            No
+          </label>
```

Comment:
add separate labels for inputs here. now when i click on `Do you have cats?`, it selects the first option(Yes)